capellambse/model/__init__.py

```diff
@@ -324,3 +328,171 @@ def search(
+            that has been used when the given Capella model was last
+            saved. After consideration of the optional placeholder this
+            function will first check if the value of *capella_cli*
+            points to a local Capella executable (that can be an
+            absolute path or an executable/ symbolic link that has been
+            made available via the environment variable ``PATH``). If no
+            executable can be found it is expected that *capella_cli*
+            represents a Docker image name for an image that behaves
+            like the Capella CLI. For the case of passing a Docker image
+            name through *capella_cli* this means it is assumed that
+            something like
+
+            .. code-block:: bash
+
+                docker run --rm -it <capella_cli> -nosplash -consolelog -app APP -appid APPID
+
+            will work.
+            The parameter *force* can be set to change the described
+            behaviour and force the function to treat the *capella_cli*
+            as a local executable or a Docker image only.
+        image_format
+            Format of the image file(s) for the exported diagram(s).
+            This can be set to any value out of ``"bmp"``, ``"gif"``,
+            ``"jpg"``, ``"png"``, or ``"svg"``.
+        create_index
+            If true two index files ``index.json`` and ``index.html``
+            will be created. The JSON file consists of a list of
+            dictionaries (each representing a diagram in the model. The
+            dictionaries come with the keys
+
+            - name: Name of the diagram as it has been set in Capella
+            - uuid: The UUID of the diagram
+            - success: A boolean stating if a diagram has been exported
+              from Capella
+
+            The HTML file showa a numbered list of diagram names which
+            are hyperlinked to the diagram image file. Right beside a
+            diagram's name one can also see the diagram's UUID in a
+            discreet light gray and tiny font size. The HTML index also
+            provides some meta data like a date time stamp for the
+            update of diagrams.
+        force
+            For the very special case that an ambiguous value is being
+            specified for *capella_cli*. It is theoretically possible to
+            specify a *capella_cli* that matches a local Capella
+            executable and an accessible Docker image in parallel. In
+            such a case the caller can force to get the value for
+            *capella_cli* interpreted as an executable with *force* set
+            to ``"exe"`` or as a Docker image when *force* is set to
+            ``"img"`` (Docker image). If *force* is not specified and an
+            ambiguous name is specified for *capella_cli*, the local
+            executable will be used and not the accessible Docker
+            image with the same name.
```

Comment:
> * Use backticks instead of asterisks for python keywords (too many to make GH suggestions).

convention is to use asterisks for parameters/ arguments of the documented function/ method. This is at least how the official Python documentation looks like. I provided one example above. Another one is here: https://raw.githubusercontent.com/python/cpython/3.11/Doc/library/logging.rst

Accepted the proposed change to the text (see c7d7a31)